main/php_syslog.h

```diff
@@ -30,26 +32,12 @@
 #endif
 #endif
 
-/*
- * The SCO OpenServer 5 Development System (not the UDK)
- * defines syslog to std_syslog.
- */
-
-#ifdef syslog
-
-#ifdef HAVE_STD_SYSLOG
-#define php_syslog std_syslog
-#endif
-
-#undef syslog
+BEGIN_EXTERN_C()
+PHPAPI void php_syslog(int, const char *message);
```

Comment:
Thanks. It's really about the API compatibility, as the even being a macro the symbol was available through the header. A simple variant could still be kept by reorganizing a bit.